android_app/app/src/main/java/com/health/openscale/core/bluetooth/BluetoothSwpSBF77.java

```diff
@@ -0,0 +1,109 @@
+/* Copyright (C) 2019  olie.xdev <olie.xdev@googlemail.com>
+ *
+ *    This program is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    This program is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with this program.  If not, see <http://www.gnu.org/licenses/>
+ */
+
+ /*
+ * Based on source-code by weliem/blessed-android
+ */
+package com.health.openscale.core.bluetooth;
+
+import static com.welie.blessed.BluetoothBytesParser.FORMAT_UINT8;
+
+import android.content.Context;
+
+import com.health.openscale.core.utils.Converters;
+import com.welie.blessed.BluetoothBytesParser;
+
+import java.util.Arrays;
+import java.util.UUID;
+
+import timber.log.Timber;
+
+public class BluetoothSwpSBF77 extends BluetoothStandardWeightProfile {
```

Comment:
Why is the class not called BluetoothSBF77? What is the abbreviation for swp?

Reply:
SWP is abbreviation for `StandardWeightProfile` in this case. But I will delete whole `BluetoothSwpSBF77` class because it SHOULD be 100% compatible with Beurer  BF105 (https://github.com/krisjans/openScale/wiki/Beurer#beurer-scales).

Reply:
After my latest changes BluetoothSBF77 and BluetoothBeurerBF105 are super-similar. But I want someone to test my version on BF105 before I delete BluetoothSBF77 and use BluetoothBeurerBF105 instead;

Reply:
@jensMF tested (and fixed bugs) my version for BF105. I renamed BluetoothSBF77 to BluetoothBeurerBF950 and used BluetoothBeurerBF105 as base class.